sematic/examples/mnist/pytorch/pipeline.py

```diff
@@ -52,10 +57,19 @@ class PipelineConfig:
     use_cuda: bool = False
 
 
+# The node selector and toleration(s) you need to use to access
+# GPUs will vary depending on your Kubernetes setup.
 GPU_RESOURCE_REQS = ResourceRequirements(
     kubernetes=KubernetesResourceRequirements(
         node_selector={"node.kubernetes.io/instance-type": "g4dn.xlarge"},
         requests={"cpu": "2", "memory": "4Gi"},
+        tolerations=[
+            KubernetesToleration(
+                key="nvidia.com/gpu",
```

Comment:
Please edit this pipeline's README that the availability of a GPU worker is required in the cluster to run it.

Reply:
done